src/librustc_mir/borrow_check/mod.rs

```diff
@@ -688,12 +688,14 @@ impl<'cx, 'tcx> dataflow::generic::ResultsVisitor<'cx, 'tcx> for MirBorrowckCtxt
         let span = term.source_info.span;
 
         match term.kind {
-            TerminatorKind::Yield { value: _, resume: _, drop: _ } if self.movable_generator => {
-                // Look for any active borrows to locals
-                let borrow_set = self.borrow_set.clone();
-                for i in flow_state.borrows.iter() {
-                    let borrow = &borrow_set[i];
-                    self.check_for_local_borrow(borrow, span);
+            TerminatorKind::Yield { value: _, resume: _, ref resume_arg, drop: _ } => {
```

Comment:
When I first looked at this commit, I wasn't sure from the commit message whether the "bitrot" here was superficial (i.e. if this was just cleaning up formatting) or if it was semantically significant.

Further reading led me to conclude that the code in its prior state .... probably didn't even build? Is that right?

If so, this commit should probably be merged backwards in the commit series, right?

Reply:
Sorry. I needed to update my changes now that `Yield` can return a value, but the `Yield` PR didn't cause a merge conflict so I only found out about it when I rebased. I'll update and fixup.